prod/westeurope/common/cdn/cdn_endpoint_assets_custom_domain/terragrunt.hcl

```diff
@@ -25,7 +25,7 @@ terraform {
 }
 
 inputs = {
-  name                = "assets"
+  name                = "assets.cdn"
```

Comment:
It's better to create a new record. Apparently assets.io.italia.it is still used

Reply:
Ok, I created a new custom domain in the same CDN Endpoint. So, now we have two CNAME records in the DNS zone pointing to the same CDN endpoint:
- `assets.io.italia.it`: the old domain kept only for backward compatibility in case some component depends on it
- `assets.cdn.io.italia.it`: the new domain

**Note**. I created a new custom domain because we need the HTTPS connection and the generated certificate (using the Azure "CDN managed certificate" capability), set the CN with the domain name (so we cannot create only an alias in the DNS, i.e. "assets" -> "cdn.assets"). By the way, all the configuration settings are shared because they are defined in the CDN endpoint. 

**Warning**. We need to remember to enable the Custom Domain HTTPS feature by hand using the Azure portal due to the following issues:
 1) azurerm_cdn_endpoint support for custom domains is still not available (see: https://github.com/terraform-providers/terraform-provider-azurerm/issues/398)
 2) due to a bug with current AZ CLI we are unable to enable https on custom domain for a CDN endpoint: the fix should be released with AZ CLI v.2.8.0, expected at 06/23/2020 (see: https://github.com/Azure/azure-cli/issues/12152)